### PR TITLE
Add resolve lock test for green gc

### PR DESCRIPTION
**Pull request by MyonKeminta**
Added a test case to check GC Resolve Lock's correctness under Green GC.

I think this PR is not being merged immediately since it uses a specific version of TiDB.

### CODE REVIEW

File: tests/resolve-lock/resolve_lock.go [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+	}
+
+	for i := 0; i < loadDataConcurrency; i++ {
+		err = <-errCh
+		if err != nil {
+			return errors.Trace(err)
+		}
+	}
+
+	// TODO: Is this the correct way to create the PD client and kv client?
```
Comment (mahjonp) [inline tests/resolve-lock/resolve_lock.go]: `nodes` only contain tidb pod, so you will get an empty set here now.

Comment (mahjonp) [reply tests/resolve-lock/resolve_lock.go]: maybe we need inject all nodes here. 😲

Comment (MyonKeminta) [reply tests/resolve-lock/resolve_lock.go]: Is there any way to access PD and TiKV?

File: Makefile [hunk 2]
```diff
@@ -19,7 +19,7 @@ all: build
 
 chaos: tidb
 
-build: fmt tidb pocket tpcc ledger txn-rand-pessimistic on-dup sqllogic block-writer \
+build: fmt tidb pocket tpcc ledger resolve_lock txn-rand-pessimistic on-dup sqllogic block-writer \
```
Comment (Illyrix) [inline Makefile]: do you mean resolve-lock?

Comment (MyonKeminta) [reply Makefile]: Sorry.

File: tests/resolve-lock/resolve_lock.go [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+			}()
+		}
+
+		key = nextScanKeyCopy
+		if len(key) == 0 {
+			break
+		}
+	}
+
+	for currentRunningRegions >= c.GenerateLockConcurrency {
```
Comment (youjiali1995) [inline tests/resolve-lock/resolve_lock.go:272]: It's useless. Do you intend to leave some goroutines running during resolving locks?

File: tests/resolve-lock/resolve_lock.go [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+				return errors.Trace(err)
+			}
+			currentRunningRegions--
+		}
+	}
+
+	return nil
+}
+
+func (c *resolveLockClient) singleLockRegion(ctx context.Context, startKey, endKey []byte) error {
```
Comment (youjiali1995) [inline tests/resolve-lock/resolve_lock.go:287]: I think only one lock is not enough...

**Comment by mahjonp** [comment]
close PR because it's duplicated with https://github.com/pingcap/tipocket/pull/202

**Pull request closed without merging**